### ISSUE TITLE
snakepit returns 404 for micromamba/win-64/latest

### ISSUE THREAD

**Issue by jvansanten** [mamba-org/mamba#1276]
I'm not sure if this is the right forum to report this, but I can't seem to get micromamba for windows at the moment:

```
> curl https://micromamba.snakepit.net/api/micromamba/win-64/latest
{"detail":"No version found for win-64/latest"}
```

linux-64 and os-arm64 appear to work. This seems to break conda-incubator/ensureconda, which relies on `https://micromamba.snakepit.net/api/micromamba/{platform}/latest` to install micromamba on demand.

**Comment by wolfv** [comment]
Hi @jvansanten yep, sorry! Our mistake. We pushed to the wrong branch on the feedstock, and then there was a caching hiccup ... I've fixed this by hard-coding `0.17.0` as `latest` right now. I hope this works again for the time being.

**Comment by jvansanten** [comment]
Great, works for me!
